### ISSUE TITLE
How to modify the style of LED widget to be the same as in CSS?

### ISSUE THREAD

**Issue by lemrose** [ControlSystemStudio/phoebus#1887]
The LED widget in Phoebus is different from the LED style in CSS. How can I adjust the LED in Phoebus to be consistent with CSS?

**Comment by kasemir** [comment]
Yes, there are subtle differences. See https://github.com/ControlSystemStudio/phoebus/wiki/Display-Builder-Compatibility
